decidim-core/lib/decidim/snippets.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
If injects the utility specific assets to the `<head>` section of the document here:
https://github.com/decidim/decidim/blob/1eba56e6439b2598131b3d15919ca85193e51d99/decidim-core/app/views/layouts/decidim/_head.html.erb#L30

It is needed because https://github.com/trailblazer/cells does not implement `content_for`:
https://github.com/trailblazer/cells/issues/308#issuecomment-120368229

And this matters because Decidim wants to [display maps in cells](https://github.com/decidim/decidim/blob/1eba56e6439b2598131b3d15919ca85193e51d99/decidim-meetings/app/cells/decidim/meetings/meetings_map/show.erb#L1).

For a bonus, this fixes a HTML validation error that you cannot include stylesheets inside the `<body>` element.